clusters/threat-actor.json

```diff
@@ -9872,7 +9816,42 @@
       },
       "uuid": "d58030e2-5673-4836-9aff-ab6d55da0bc0",
       "value": "SLIME29"
+    },
+    {
+      "description": "Goblin Panda is one of a handful of elite Chinese advanced persistent threat (APT) groups. Most Chinese APTs target the United States and NATO, but Goblin Panda focuses primarily on Southeast Asia.",
+      "meta": {
+        "attribution-confidence": "75",
+        "cfr-suspected-state-sponsor": "China",
+        "cfr-suspected-victims": [
+          "Malaysia",
+          "India",
+          "Indonesia",
+          "Japan",
+          "Philippines",
+          "Southeast Asia",
+          "South Korea",
+          "Vietnam"
+        ],
+        "cfr-target-category": [
+          "Private Sector"
+        ],
+        "country": "CN",
+        "refs": [
+          "https://www.crowdstrike.com/blog/meet-crowdstrikes-adversary-of-the-month-for-august-goblin-panda/",
+          "https://securelist.com/cycldek-bridging-the-air-gap/97157/",
+          "https://www.fortinet.com/blog/threat-research/cta-security-playbook--goblin-panda.html",
+          "https://go.crowdstrike.com/rs/281-OBQ-266/images/Report2020CrowdStrikeGlobalThreatReport.pdf",
+          "https://cyberthreat.thalesgroup.com/sites/default/files/2022-05/THALES%20THREAT%20HANDBOOK%202022%20Light%20Version_1.pdf"
+        ],
+        "synonyms": [
+          "Conimes",
+          "Cycldek",
+          "ATK34"
```

Comment:
ATK34 overlaps with APT30, not GOBLIN PANDA.
See 1st bullet of https://github.com/MISP/misp-galaxy/pull/749#issuecomment-1217452683

Reply:
Thales says it overlaps with Golbin Panda page 53 here: https://cyberthreat.thalesgroup.com/sites/default/files/2022-05/THALES%20THREAT%20HANDBOOK%202022%20Light%20Version_1.pdf
<img width="241" alt="Screen Shot 2022-08-17 at 1 53 37 PM" src="https://user-images.githubusercontent.com/23531109/185241033-377f1bf1-7897-419e-8274-9ff900f4a5b8.png">

Reply:
On another page, they say it is APT30. Weird
![Screenshot_20220818-103923~2.png](https://user-images.githubusercontent.com/49360849/185298757-0d3d1389-cb46-407d-8982-bbeeb450e2da.png)

Reply:
Oh
I'll just remove that alias then, is it ok?

Reply:
For now ok. Let me ask someone from Thales.